3_3.kut
Sprawdzanie typ√≥w.
Test przechodzi pozytywnie.
###
fn add(a: int, b: int) {
    a + b
}

println(add(1, 2))
println(add("hello", "world"))
###
3
Cannot convert 'hello' to int
